### PR TITLE
Tweaks rupee's in-game sprite and adds some failtext with IDs.

### PR DESCRIPTION
**Pull request by ArcaneMusic**
## About The Pull Request

In the rare, off chance that a player manages to get sent to the super secret room, this tweaks the sprite to look juuuuust a bit better and glossier like a real gemstone. In addition, this adds a bit of splashtext to the Id card if a player attempts to use it like cash or credits.
image

## Why It's Good For The Game

I was recently given some tips on making crystalline sprites and I recalled how old these looked, so this was an easy sprite tweak. Also, I wanted to make sure that the interactions between rupees and cash existed in a clear and concise way, so I added a quick early return comment to get the full effect.

## Changelog
:cl:
imageadd: Some hidden debug content got some new sprites, should you ever find it.
/:cl:

### CODE REVIEW

**Comment by maxymax13** [comment]
these look way too noisy

**Review by Twaticus — APPROVED** [review]
ok

**Comment by cacogen** [comment]
they should be worth the amount they are in lizard of zelda

File: code/game/objects/items/cards_ids.dm [hunk 1]
```diff
@@ -423,7 +423,10 @@
 					SSid_access.apply_trim_to_card(src, trim)
 
 /obj/item/card/id/attackby(obj/item/W, mob/user, params)
-	if(istype(W, /obj/item/holochip))
+	if(istype(W, /obj/item/rupee))
+		to_chat(user, "<span class='warning'>Your ID smartly rejects the strange shard of glass. Who knew, apparently it's not ACTUALLY valuable!</span>")
```
Comment (Mothblocks) [inline code/game/objects/items/cards_ids.dm:427]: all this stuff should eventually be signals but i'm feeling nice today

**Pull request merged**